src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -39,12 +45,17 @@ public void startExecuting()
             return;
         }
 
-        citizen.getNavigator().tryMoveToXYZ((double) pos.getX() + 0.5D, (double) pos.getY(), (double) pos.getZ() + 0.5D, 1.0D);
-    }
+        if (citizen.getWorkBuilding() != null)
+        {
+            /*
+            Temp fix for pathfinding in the night.
+            Citizens can't find a path home.
+             */
+            final BlockPos workBuilding = citizen.getWorkBuilding().getLocation();
```

Comment:
Yeah I did want to change that. Current pathfinder is a mess sadly. And workHut is the best compromise

Am 06.09.2016 16:58 schrieb Colton Schlosser <notifications@github.com>:

In src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java:

> @@ -39,12 +45,17 @@ public void startExecuting() > return; > } > > - citizen.getNavigator().tryMoveToXYZ((double) pos.getX() + 0.5D, (double) pos.getY(), (double) pos.getZ() + 0.5D, 1.0D); > - } > + if (citizen.getWorkBuilding() != null) > + { > + /* > + Temp fix for pathfinding in the night. > + Citizens can't find a path home. > + */ > + final BlockPos workBuilding = citizen.getWorkBuilding().getLocation(); 

Did you want to change from getHomePosition to getWorkBuilding?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.